libs/commons/Platform.mli

```diff
@@ -0,0 +1,5 @@
+(* The machine architecture, such as arm64, x86_64 *)
```

Comment:
woud probably be better to define a proper enums for the
arch and for the kernel.
In fact this is partly done in Sys.mli, so I wonder why we don;t use that. At least
add a TODO saying we should maybe use Sys.os_type